velox/dwio/dwrf/common/FileMetadata.cpp

```diff
@@ -90,8 +90,9 @@ TypeKind TypeWrapper::kind() const {
       return TypeKind::ROW;
     case proto::orc::Type_Kind_VARCHAR:
       return TypeKind::VARCHAR;
+    // Date is an int32 for the number of days since EPOCH ?
```

Comment:
@aditi-pandit The max value is 2147483647 of int32, and it can represent time ”2038-01-19 11:14:07". If the time is greater than ”2038-01-19 11:14:07", how will we represent it with int32?

Reply:
@jiangjiguang this is `Date` and not `Time`. This can approximately represent `(2147483647 / 365) ~ 5883516` years from the UNIX epoch.
There is no support for Time yet.